main.go

```diff
@@ -14,21 +14,47 @@ import (
 	"os"
 	"os/user"
 	"path/filepath"
+	"strings"
 	"time"
 
+	"github.com/dustin/go-humanize"
 	"github.com/marcopaganini/logger"
 )
 
 const (
 	stateFile          = ".testmynet-cli.state"
-	minDurationMinutes = 15
```

Comment:
Oh, the 15 minutes was also a number that came out of testmy.net (IIRC), so it should stay.